packages/web/app/src/pages/organization-settings.tsx

```diff
@@ -709,3 +728,347 @@ export function DeleteOrganizationModalContent(props: {
+                                  )?.user.displayName
+                                : 'Select new owner'}
+                              <ChevronsUpDown className="ml-2 size-4 shrink-0 opacity-50" />
+                            </Button>
+                          </PopoverTrigger>
+                          <PopoverContent className="max-h-[--radix-popover-content-available-height] w-[--radix-popover-trigger-width] p-0">
+                            <Command>
+                              <CommandInput placeholder="Search member..." />
+                              <CommandEmpty>No members found</CommandEmpty>
+                              <CommandGroup>
+                                {props.members.map(member => (
+                                  <CommandItem
+                                    key={member.id}
+                                    {...field}
+                                    className="cursor-pointer"
+                                    value={
+                                      props.members.find(
+                                        member => member.user.email === selected?.user.email,
+                                      )?.user.displayName
+                                    }
+                                    onSelect={() => {
+                                      onSelect(member);
+                                      setOpenPopover(false);
+                                    }}
+                                  >
+                                    <Check
+                                      className={cn(
+                                        'mr-2 size-4',
+                                        selected === member ? 'opacity-100' : 'opacity-0',
+                                      )}
+                                    />
+                                    <div className="flex flex-row gap-1">
+                                      <span className="font-bold">{member.user.displayName}</span>
+                                      <span>-</span>
+                                      <span className="text-gray-500">({member.user.email})</span>
+                                    </div>
+                                  </CommandItem>
+                                ))}
+                              </CommandGroup>
+                            </Command>
+                          </PopoverContent>
+                        </Popover>
+                      </FormControl>
+                      <FormMessage />
+                    </FormItem>
+                  );
+                }}
+              />
+
+              <FormField
+                control={props.form.control}
+                name="confirmation"
+                render={({ field }) => {
+                  return (
+                    <FormItem>
+                      <FormLabel>Type {props.organization.cleanId} to confirm</FormLabel>
+                      <FormControl>
```

Comment:
Taking UX from GitHub to disallow users selecting the org name from copy/pasting. This way we are more strict with the action of transferring orgs

```suggestion
                      <FormLabel className="select-none">
```